src/modules/addScore.js

```diff
@@ -1,31 +0,0 @@
-import Score from './scores.js';
-
-const AddScore = () => {
-  const name = document.getElementById('name');
-  const score = document.getElementById('score');
-  const index = Score.scoreList.length + 1;
-  if (name.value === '' || score.value === '') {
-    return null;
-  }
-  const newList = new Score(name.value, score.value, index);
-  Score.scoreList.push(newList);
-  name.value = '';
-  score.value = '';
-  return Score.scoreList.length;
-};
-
-const Render = () => {
-  const leaderBox = document.querySelector('.leaderboard-box');
-  if (Score.scoreList.length === 0) {
-    leaderBox.innerHTML = 'No new Score Added';
-  } else {
-    let html = '';
-    Score.scoreList.forEach((score) => {
-      html += `<li class="score"><p class="name">${score.name}: ${score.score}</p></li>`;
-    });
-    leaderBox.innerHTML = html;
-  }
-  return Score.scoreList.length;
-};
-
-export { AddScore, Render };
```

Comment:
- [ ] Let's keep our codebase clean. Kindly remove this empty and unused file here.

  ![nico-add](https://user-images.githubusercontent.com/36542986/151272588-6bb9076d-273c-4c06-8375-41535a826f80.jpg)

Reply:
Oh.. sorry for that I did not check it 😢. Thanks for your feedback 👍